src/main/java/nl/tudelft/dnainator/ui/widgets/AboutDialog.java

```diff
@@ -0,0 +1,83 @@
+package nl.tudelft.dnainator.ui.widgets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import javafx.scene.control.ButtonBar.ButtonData;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Stage;
+
+/**
+ * Creates a {@link Dialog} displaying information about the
+ * application.
+ */
+public class AboutDialog {
+	private Dialog<String> dialog;
```

Comment:
For our other dialogs, we use the newer [Alert](http://download.java.net/jdk9/jfxdocs/javafx/scene/control/Alert.html). You might consider using this as well